.devcontainer/devcontainer.json

```diff
@@ -9,11 +9,17 @@
         "terraform": {
             "version": "latest",
             "tflint": "none",
-            "terragrunt": "none"
-        }
+            "terragrunt": "latest"
+        },
+        "github-cli": "latest"
     },
+    "mounts": [
+        "source=${env:HOME}/.ssh,target=/home/vscode/.ssh,type=bind,consistency=cached",
```

Comment:
I need to comment out line 17 to be able to access my SSH configs from within the container.